cli/src/movement_util.rs

```diff
@@ -25,16 +25,24 @@ use crate::cli_util::{short_commit_hash, WorkspaceCommandHelper};
 use crate::command_error::{user_error, CommandError};
 use crate::ui::Ui;
 
-pub enum Direction {
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub(crate) struct MovementArgs {
+    pub(crate) offset: u64,
+    pub(crate) edit: bool,
+    pub(crate) conflict: bool,
```

Comment:
nit: `(crate)` can be omitted since outer type is `pub(crate)`.

Reply:
I didn't know that. Done. Thanks.